source/html/js/app/notes.js

```diff
@@ -67,18 +67,18 @@ let promise_delete_closure = (endpoint, api_key) => {
 };
 
 const delete_resource_notes = function (arn) {
-    var current_connection = connections.get_current();
-    var url = current_connection[0];
-    var api_key = current_connection[1];
-    var current_endpoint = `${url}/notes/${encodeURIComponent(arn)}`;
+    let current_connection = connections.get_current();
+    let url = current_connection[0];
+    let api_key = current_connection[1];
+    let current_endpoint = `${url}/notes/${encodeURIComponent(arn)}`;
     return new Promise(promise_delete_closure(current_endpoint, api_key));
 };
 
 const delete_all_resource_notes = function () {
-    var current_connection = connections.get_current();
-    var url = current_connection[0];
-    var api_key = current_connection[1];
-    var current_endpoint = `${url}/notes`;
+    let current_connection = connections.get_current();
```

Comment:
Some of these seem like they could be const instead of let

Reply:
same as the other comment